src/renderer/styles/app.global.scss

```diff
@@ -48,8 +48,9 @@ a {
   }
 
   &.bp4-disabled {
+    background-color: var.$c-green-40;
+    cursor: not-allowed;
```

Comment:
I'm OK with this change, but I have a thought for the future: If `not-allowed` is not a default for disabled buttons, perhaps it's meant to be that way? And if we want to change it, it would be good to change it for *all* disabled buttons, so that our UX is consistent.

This applies to all custom styling: it's safer to stay with the defaults and if we really want to change them - we should be consistent.

Reply:
This change affects all 'primary' buttons, but I guess there are some not having 'primary' class. I'll remove it to have a simple pointer cursor